tests/fixtures/test1.json

```diff
@@ -1 +1 @@
-{"airport":{"height":15,"width":15,"x":0,"y":43,"pixelRatio":1},"my_bbq":{"height":32,"width":32,"x":43,"y":0,"pixelRatio":1},"airport_xl":{"height":43,"width":43,"x":0,"y":0,"pixelRatio":1}}
\ No newline at end of file
+{"airport":{"height":15,"width":15,"x":0,"y":43,"pixelRatio":1,"sdf":false},"my_bbq":{"height":32,"width":32,"x":43,"y":0,"pixelRatio":1,"sdf":false},"airport_xl":{"height":43,"width":43,"x":0,"y":0,"pixelRatio":1,"sdf":false}}
```

Comment:
I just realised this.

I think spritezero only add `sdf` property if SDF option is used
Is it possible to remove `sdf` field from json?

Reply:
I remove `sdf` field from json.